lib/versioning/rez/index.ts

```diff
@@ -67,8 +67,8 @@ function isGreaterThan(a: string, b: string): boolean {
 
 function isLessThanRange(version: string, range: string): boolean {
   return (
-    npm.isVersion(padZeroes(version)) &&
-    npm.isLessThanRange(padZeroes(version), rez2npm(range))
+    !!npm.isVersion(padZeroes(version)) &&
```

Comment:
```suggestion
    npm.isVersion(padZeroes(version)) &&
```

Reply:
Not required, as npm also only returns a boolean

Reply:
Ups, read wrong,you are right 👍